appinfo/Migrations/Version20170801085340.php

```diff
@@ -11,6 +11,11 @@ class Version20170801085340 implements ISchemaMigration {
 	public function changeSchema(Schema $schema, array $options) {
 		$prefix = $options['tablePrefix'];
 		if ($schema->hasTable("{$prefix}notifications")) {
+			// fix migration from 8.2.11 to 10.0.x.
+			// expected datatype for the column is string not integer
+			$table = $schema->getTable("{$prefix}notifications");
+			$column = $table->getColumn('object_id');
+			$column->setType(Type::getType(Type::STRING));
```

Comment:
- set only if current type is integer?
- do we need a new migration? there might be situations where this migration is already executed but the typ is still wrong

Reply:
> set only if current type is integer?

Ok

> do we need a new migration? there might be situations where this migration is already executed but the typ is still wrong

I'll add a new migration for safety reasons. It's weird that installations post 8.2.11 don't have the problem fixed even manually, but we can't ensure it.